test/specs/luxon-adapter.spec.js

```diff
@@ -0,0 +1,39 @@
+describe('Luxon Adapter', function() {
```

Comment:
this file should use tabs for consistency? and we should probably setup the linter to lint the tests since it wasn't caught

Reply:
Oops, yes. Thought I'd converted them all I but I guess I missed this one.

Reply:
I also added the tests to the lint config and resolved the issues it raised.